bb_wrapper/wrapper/bb.py

```diff
@@ -16,6 +16,8 @@ class BaseBBWrapper(RequestsWrapper):
 
     SCOPE = ""
 
+    UNAUTHORIZED = [401, 403]
+
```

Comment:
Tem que armazenar os dados da autenticação em um storage compartilhado entre as diversas instâncias!

Se não fizer isso, a cada nova instância do wrapper será necesário se reautenticar!

```suggestion
    data = threading.local()
```